src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -151,6 +151,11 @@ void CreateInteractionChoiceSetRequest::Run() {
     // we have VR commands
     SendVRAddCommandRequests(app);
   } else {
+    if (vr_status == MessageHelper::ChoiceSetVRCommandsStatus::NONE) {
```

Comment:
@IGapchuk swap comparation

Reply:
@LuxoftAKutsan Changed in 25344b813009d66f7c943e1451025504b80f44ab.